sw/device/tests/otbn/otbn_rsa_test.c

```diff
@@ -152,11 +154,15 @@ static void rsa_decrypt(otbn_t *otbn_ctx, const uint8_t *modulus,
   CHECK(otbn_ctx != NULL);
   CHECK(size_bytes % 32 == 0);
 
+  uint32_t mode = 2;
```

Comment:
It seems preferred to push this line down to the first use, i.e. right before line 164 now. And maybe add a comment `// decrypt` to it?